crates/astria-sequencer/src/sequencer.rs

```diff
@@ -21,6 +21,9 @@ use crate::{
     state_ext::StateReadExt as _,
 };
 
+// TODO: use penumbra's `IBC_SUBSTORE_PREFIX` after they merge #3419
```

Comment:
if they merge this before this PR is merged (likely), i will update this PR, otherwise i'll do a follow-up.